layouts/default-max.vue

```diff
@@ -12,9 +24,36 @@
 <script lang="ts">
 import Vue from 'vue'
 import { Component } from 'vue-property-decorator'
+import QiskitBanner from '@qiskit-community/qiskit-vue/src/components/banner/Banner.vue'
 
-@Component
+@Component({
+  components: { QiskitBanner }
+})
 export default class MaxLayout extends Vue {
   isMenuShown: boolean = false
 }
 </script>
+
+<style lang="scss">
+@import "@carbon/colors/scss/colors";
+
+.qiskit-banner.qiskit-runtime {
+  background: $background-color-lighter !important;
```

Comment:
super tiny request -- should this be specified to `background-color`

Reply:
👍🏽